core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -679,4 +755,33 @@ private ExprValue exprYear(ExprValue date) {
     return new ExprIntegerValue(date.dateValue().getYear());
   }
 
+  /**
+   * NOW() returns a constant time that indicates the time at which the statement began to execute
```

Comment:
`statement` or `query`?

Reply:
Copied from MySQL doc: https://dev.mysql.com/doc/refman/8.0/en/date-and-time-functions.html#function_now.